os/booting-on-vmware.md

```diff
@@ -45,11 +63,20 @@ The last step uploads the files to your ESXi datastore and registers your VM.
 You can now tweak the VM settings, like memory and virtual cores, then power it
 on. These instructions were tested to deploy to an ESXi 5.5 host.
 
+### Booting with VMware Workstation 12
+
+Run VMware Workstation GUI:
+
+1. in the menu, click "File > Open..."
+2. in the wizard, specify the location of the OVA downloaded earlier
+3. name your VM and then click "Import"
+4. press "Retry" if VMware Workstation has raised "OVF specification" warning
+5. edit VM settings if necessary
+6. start up your CoreOS VM
+
 ## Cloud-Config
 
-Cloud-config can be specified by attaching a [config-drive][config-drive] with
-the filesystem label `config-2`. This is commonly done through whatever
-interface allows for attaching CD-ROMs or new drives.
+Cloud-config can be specified by attaching a [config-drive][config-drive] with the filesystem label `config-2`. This is commonly done through whatever interface allows for attaching CD-ROMs or new drives. In addition in CoreOS 801.0.0 and greater you can use [VMware GuestInfo][vmware backdoor].
```

Comment:
Based on the trouble that everyone has with the config-drive, should we just drop this altogether and rely on the guest info stuff?